composer.json

```
@@ -22,7 +22,7 @@
         "php": "^7.2 || ^8.0",
         "ext-json": "*",
         "composer-plugin-api": "^1.1 || ^2.0",
-        "friendsofphp/php-cs-fixer": "^2.16.5",
+        "friendsofphp/php-cs-fixer": "^2.16||^3.0",
```

That specific patch request was there for a reason, see #26 
```suggestion
        "friendsofphp/php-cs-fixer": "^2.16.5||^3.0",
```